skglm/tests/test_prox_newton.py

```diff
@@ -0,0 +1,63 @@
+import pytest
+import numpy as np
+from numpy.linalg import norm
+
+from sklearn.linear_model import LogisticRegression as LogReg_sklearn
+
+from skglm.penalties import L1, L1_plus_L2
+from skglm.datafits import Logistic
+from skglm.solvers.cd_solver import cd_solver
+from skglm.solvers.prox_newton_solver import prox_newton_solver
+
+from skglm.utils import make_correlated_data
+
+
+n_samples, n_features = 30, 50
+X, y, _ = make_correlated_data(
+    n_samples=n_samples, n_features=n_features, density=0.1, random_state=0)
+y_ind = np.sign(y)
+
+np.random.seed(0)
+
+alpha_max = norm(X.T @ y, ord=np.inf) / n_samples
```

Comment:
```suggestion
alpha_max = norm(X.T @ y, ord=np.inf) / (2 * n_samples)
```

``alpha_max`` for logreg is not the same as quad datafit.
in general, it has the following formula  $\\Omega^D(X^T \\nabla F(0))$
(cf. [Gap Safe screening rules for sparsity enforcing penalties](https://arxiv.org/abs/1611.05780) proposition 4)

In our case $\\nabla F(0) = \\frac{1}{2n} y$

Can you add unitest to check that we get a zero solution when we are at ``alpha_max``